docs/theming/component-variables.mdx

```diff
@@ -534,6 +534,54 @@ Defined in: [https://github.com/GetStream/stream-chat-css/blob/main/src/v2/style
 
 Defined in: [https://github.com/GetStream/stream-chat-css/blob/main/src/v2/styles/ImageCarousel/ImageCarousel-theme.scss](https://github.com/GetStream/stream-chat-css/blob/main/src/v2/styles/ImageCarousel/ImageCarousel-theme.scss)
 
+## Notification list
+
+| Name                                                | Description                                             |
+| --------------------------------------------------- | ------------------------------------------------------- |
+| `--str-chat__notification-list-font-family`         | The font used in the component                          |
+| `--str-chat__notification-list-color`               | The text/icon color of the component                    |
+| `--str-chat__notification-list-background-color`    | The background color of the component                   |
+| `--str-chat__notification-list-border-radius`       | The border radius used for the borders of the component |
+| `--str-chat__notification-list-border-block-start`  | Top border of the component                             |
+| `--str-chat__notification-list-border-block-end`    | Bottom border of the component                          |
+| `--str-chat__notification-list-border-inline-start` | Left (right in RTL layout) border of the component      |
+| `--str-chat__notification-list-border-inline-start` | Right (lrft in RTL layout) border of the component      |
```

Comment:
```suggestion
| `--str-chat__notification-list-border-inline-end` | Right (left in RTL layout) border of the component      |
```
Typo

Reply:
Will fix this after the ongoing PRs merged into theming-v2